languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -31941,10 +31941,11 @@ though, and has since been slightly modified)-->
                 <token>/</token>
                 <token>ou</token>
             </pattern>
-            <message>Sauf dans des contextes très techniques, cette expression est souvent redondante. Considérez n'utiliser que l'un ou l'autre.</message>
+            <message>Cette expression signifiant "et ou ou" est redondante et peut être simplifiée pour apporter de la fluidité à votre texte.</message>
```

Comment:
That's corrected. Indeed in extremely particular cases it might make sense to put this expression but it is not pretty. For proper writting, I would always suggest to reformulate and avoid it.